python/pyspark/errors/error-conditions.json

```diff
@@ -507,6 +507,11 @@
       "Variant binary is malformed. Please check the data source is valid."
     ]
   },
+  "MASTER_URL_INVALID": {
```

Comment:
shouldn't we already have a error condition for it? How do we validate MASTER url before without Spark Connect?

Reply:
We actually don't have the error class (and even if there is, that has to be listed in JVM's error class JSON file). The error is thrown from `prepareSubmitEnvironment` at `core/src/main/scala/org/apache/spark/deploy/SparkSubmit`.